simpleSequential/test_addTwoNumericalString.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+from addTwoNumericalString import addTwoNumericalString
+
+class TestAddTwoNumericalString(unittest.TestCase):
```

Comment:
The whole test did not do what the problem says.

Reply:
The whole test still not do what the problem says. Please visit my example that I gave in the problem statements.